test/expect/HasDecompTest.test_aten_core_operators.expect

```diff
@@ -447,8 +447,6 @@ aten::sinh
 aten::sinh.out
 aten::sinh_
 aten::slice.Tensor
-aten::special_bessel_j0
-aten::special_bessel_j0.out
```

Comment:
You should need to rerun the expectest here, since you removed the decomp from the core ATen list.

Reply:
Fixed. Thanks for the hint!